src/create_pr.rs

```diff
@@ -29,12 +29,20 @@ pub async fn run() -> Result<(), CreateError> {
         .await?;
 
     let target = inputs::get_target_branch(branches)?;
+    
+    let diff = github::get_diff(&git_info.branch, &target)?;
+    if diff.trim().is_empty() {
+        return Err(CreateError::EmptyDiff(git_info.branch.clone(), target.clone()));
```

Comment:
avoid manually throwing here but rather just throw inside of `get_diff` in case of an empty diff